docs/hub/diffusers.md

```diff
@@ -0,0 +1,49 @@
+# Using 🧨 `diffusers` at Hugging Face
+
+🧨 `diffusers` is the go-to library for state-of-the-art pretrained diffusion models for generating images, audio, and even 3D structures of molecules. Whether you’re looking for a simple inference solution or want to train your own diffusion model, 🤗 Diffusers is a modular toolbox that supports both. Our library is designed with a focus on usability over performance, simple over easy, and customizability over abstractions.
+
+## Exploring 🧨 diffusers in the Hub
+
+There are over 10,000 `diffusers` compatible pipelines on the Hub which you can find by filtering at the left of [the models page](https://huggingface.co/models?library=diffusers&sort=downloads). Diffusion systems are typically composed of multiple components such as text encoder, UNet, VAE, and scheduler. This is why they are not standalone models. 
+
+You can find diffusion pipelines for many different tasks:
+
+* Generating images from natural language text prompt ([text-to-image](https://huggingface.co/models?library=diffusers&pipeline_tag=text-to-image&sort=downloads)).
+* Translating images from images using natural language text prompt ([image-to-image](https://huggingface.co/models?library=diffusers&pipeline_tag=image-to-image&sort=downloads)).
+* Generating videos from natural language text prompt ([text-to-video](https://huggingface.co/models?library=diffusers&pipeline_tag=text-to-video&sort=downloads)).
```

Comment:
WDYT about this minor rewording?

```suggestion
* Generating images from natural language text prompts ([text-to-image](https://huggingface.co/models?library=diffusers&pipeline_tag=text-to-image&sort=downloads)).
* Creating new images based on input images and text prompts ([image-to-image](https://huggingface.co/models?library=diffusers&pipeline_tag=image-to-image&sort=downloads)).
* Generating videos from text prompts ([text-to-video](https://huggingface.co/models?library=diffusers&pipeline_tag=text-to-video&sort=downloads)).
```

Reply:
I wrote my suggestions before reading Omar's, his works for me too.